Amazon.Ion.ObjectMapper.Test/IonObjectSerializerTest.cs

```diff
@@ -58,6 +58,7 @@ public void DeserializesObjectsWithIgnoreReadOnlyFields()
             Assert.IsNull(deserialized.lastName);
             Assert.IsNotNull(deserialized.department);
             Assert.IsNull(deserialized.birthDate);
+            Assert.IsNotNull(deserialized.car);
```

Comment:
This change seems unrelated. Please move it to another PR.

Reply:
Removed car from Teacher class so this line is removed.